models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,104 @@
+# mobilefacedet-v1-mxnet
+
+## Use Case and High-Level Description
+
+  MobileFace Detection V1 is a Light and Fast Face Detector for Edge Devices (LFFD) model based on Yolo V3 architecture and trained with MXNet\*. For details see the [repository](https://github.com/becauseofAI/MobileFace) and [paper](https://arxiv.org/pdf/1904.10633.pdf).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.5456       |
+| MParams           | 7.6828        |
+| Source framework  | MXNet\*       |
+
+## Input
+
+### Original model
+
+Image, name - `data`, shape - `[1x256x256x3]`, format -`[BxHxWxC]` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Expected color order -  `BGR`.
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+> **WARNING:** Please pay attention that input layout of converted model is `[BxHxWxC]`.
```

Comment:
```suggestion
> **WARNING:** Please note that the input layout of the converted model is `[BxHxWxC]`.
```

You can only pay attention _to_ something, not _that_ something happens. It's possible to contort the sentence to keep "pay attention" in it (e.g. "pay attention to the fact that the input layout is [...]"), but it would be simpler to just use "note".

Reply:
Done